tests/dummy/app/templates/components/login-form.hbs

```diff
@@ -7,14 +7,14 @@
 <form {{action 'authenticateWithOAuth2' on='submit'}}>
   <div class="form-group">
     <label for="identification">Login</label>
-    {{input value=identification placeholder='Enter Login' class='form-control'}}
+    <input type="text" placeholder="Enter Login" class="form-control" onchange={{action 'updateIdentification'}} value={{this.identification}} />
```

Comment:
hmm, what's the reason for this change? why not keep using the `<Input>` component?

Reply:
I find using native inputs easier to follow and more explicit. What would be an advantage of the `<Input>` component?